### PR TITLE
Fix runtime in timed go_bald

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -885,6 +885,8 @@
 	return 1
 
 /datum/species/proc/go_bald(mob/living/carbon/human/H)
+	if(QDELETED(H))	//may be called from a timer
```
Comment (optimumtact) [inline code/modules/mob/living/carbon/human/species.dm]: aren't timers supposed to check this?

Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species.dm]: Timers check `src`

Comment (optimumtact) [inline code/modules/mob/living/carbon/human/species.dm]: they should check args I reckon but that's another pr

Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species.dm]: `null` is a valid arg though, see #25683

**Pull request merged**